src/test/java/uk/gov/hmcts/reform/cwrdapi/TestSupport.java

```diff
@@ -86,6 +94,144 @@ public static List<CaseWorkerDomain> buildSuspendedCaseWorkerProfileData() {
                 .build());
     }
 
+
+    public static boolean validateSearchUserProfileResponse(ResponseEntity<List<SearchStaffUserResponse>> response,
+                                                            SearchRequest searchReq) {
+
+        SearchRequest finalSearchReq = searchReq;
+        List validResponse = new ArrayList<>();
```

Comment:
change it to List<SearchStaffUserResponse>

Reply:
changes done